renderlib/command.cpp

```diff
@@ -769,6 +768,39 @@ SetClipPlaneCommand::execute(ExecutionContext* c)
   c->m_renderSettings->m_DirtyFlags.SetFlag(RenderParamsDirty);
 }
 
+void
+SetColorRampCommand::execute(ExecutionContext* c)
```

Comment:
any new command has to provide a `execute`, `parse` , `write`, and `toPythonString` implementation